CMakeLists.txt

```diff
@@ -18,53 +25,33 @@ if (NOT ASSIMP_FOUND)
   set(ASSIMP_INCLUDE_DIRS ${ASSIMP_PC_INCLUDE_DIRS})
 endif()
 
+find_package(shape_msgs REQUIRED)
+find_package(rclcpp REQUIRED)
 # find *absolute* paths to ASSIMP_LIBRARIES
 # Both, pkg-config and assimp's cmake-config don't provide an absolute library path.
 # For, pkg-config the path is in ASSIMP_PC_LIBRARY_DIRS, for cmake in ASSIMP_LIBRARY_DIRS.
 find_library(ASSIMP_ABS_LIBRARIES NAMES ${ASSIMP_LIBRARIES} assimp HINTS ${ASSIMP_LIBRARY_DIRS} ${ASSIMP_PC_LIBRARY_DIRS})
 set(ASSIMP_LIBRARIES "${ASSIMP_ABS_LIBRARIES}")
-
 find_package(Boost REQUIRED system filesystem)
-
 find_package(console_bridge REQUIRED)
-
+find_package(eigen3_cmake_module REQUIRED)
```

Comment:
Is this really needed?

Reply:
Unfortunately, yes. Otherwise headers are not found and export doesn't work. See
 https://github.com/ros2/eigen3_cmake_module